.buildkite/test-pipeline.yaml

```diff
@@ -47,7 +47,7 @@ steps:
     - pytest -v -s prefix_caching
 
 - label: Samplers Test
-  command: pytest -v -s samplers --forked
```

Comment:
without this, there were some weird failures related to fork. I assume there are flakiness here

Reply:
This may be fixed by this PR btw https://github.com/vllm-project/vllm/pull/3487#issuecomment-2005780263